stix_shifter_utils/utils/helpers.py

```diff
@@ -1,16 +1,11 @@
 import json
-
-class StixObjectIdEncoder(json.JSONEncoder):
-    def default(self, obj):
-        if isinstance(obj, StixObjectId):
-            return obj.object_id
-        # Let the base class default method raise the TypeError
-        return json.JSONEncoder.default(self, obj)
+import json_fix
 
 
 class StixObjectId(object):
     object_id:str = None
     def __init__(self, object_id):
+        json_fix
```

Comment:
any initialization value `json_fix` == ?

Reply:
yes, it is a hack to make EDI and other developer (not to delete it when EDI suggests it is not used) to think it is used somehow, not just `import json_fix`
in reality just import json_fix is enough

Reply:
having a second thought on this, I think you are right, any developer that sees this hack will think it is not initialized variable
removed, added comment next to import